deb/common/control

```diff
@@ -39,7 +36,7 @@ Recommends: apparmor,
             pigz,
             procps,
             xz-utils
-Suggests: aufs-tools [amd64], cgroupfs-mount | cgroup-lite
```

Comment:
Double checked if it wasn't there for some other obscure purpose, but this was originally added as part of;

- https://github.com/moby/moby/pull/816

Reply:
☝️ looks like there's potentially some other bits left behind in moby related to AuFS; I see [`auplink`](https://manpages.debian.org/jessie/aufs-tools/auplink.8.en.html) in there, and it looks like we still account for that in our AppArmor template; https://github.com/moby/moby/blob/7faa4ecfbfe5f641578931ff120c6e0f8faae16a/contrib/apparmor/template.go#L118-L136